go/enclave/db/rawdb/accessors_chain.go

```diff
@@ -136,13 +139,17 @@ func ReadBodyRLP(db ethdb.KeyValueReader, hash gethcommon.Hash, number uint64, l
 	return data
 }
 
-func ReadRollupsForHeight(db ethdb.Database, number uint64, logger gethlog.Logger) []*core.Rollup {
+func ReadRollupsForHeight(db ethdb.Database, number uint64, logger gethlog.Logger) ([]*core.Rollup, error) {
 	hashes := ReadAllHashes(db, number)
 	rollups := make([]*core.Rollup, len(hashes))
 	for i, hash := range hashes {
-		rollups[i] = ReadRollup(db, hash, logger)
+		rollup, err := ReadRollup(db, hash, logger)
```

Comment:
If the rollup is not found should it `rollups[i] = nil` ?

Reply:
I think it should all succeed or all fail.